webview/setup-tests.js

```diff
@@ -15,4 +15,13 @@ const mutationObserverMock = jest.fn().mockImplementation(() => {
 })
 global.MutationObserver = mutationObserverMock
 
+const intersectionObserverMock = jest.fn().mockImplementation(() => {
+  return {
+    disconnect: jest.fn(),
+    observe: jest.fn(),
+    unobserve: jest.fn()
+  }
+})
+global.IntersectionObserver = intersectionObserverMock
```

Comment:
Could be made better if we wanted to actually puppet the observer, but I think we have better ways to test most scenarios it would apply to.